### PR TITLE
Quiz Page and Authentication Pages UI

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added Quiz Starting page, Questions page, Login Page ,Sign Up page and profile page.
Files to check:
Category.jsx
Quiz.jsx
QuizQuestions.jsx
Login.jsx
Signup.jsx
Profile.jsx

// commented code in auth file are to be used later

### CODE REVIEW

**Review by Kevin-Solomon — APPROVED** [review]

File: src/components/CategoryCard/CategoryCard.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+.category-card img {
+	height: 25rem;	
+      border-radius: 10px;
+}
+.category-card-text {
+	position: absolute;
+	bottom: 1.6rem;
+	left: 1.6rem;
+}
```
Comment (Dev-AkshatGupta) [inline src/components/CategoryCard/CategoryCard.css:20]: code is nicely indented

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
 footer {
 	background-color:var(--hover-color);
 	padding: 1.6rem;
+	/* margin-top: 10rem; */
```
Comment (Dev-AkshatGupta) [inline src/components/Footer/Footer.css]: Suggestion-remove these commented codes

File: src/pages/authentication/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      password: signUpFormState.password,
+      name: signUpFormState.name,
+    };
+    console.log(post_obj);
+    //     const response = await signupuser(post_obj);
+    //     if (response.success) {
+    //       navigate("/login");
+    //     } else {
+    //       console.log("SOME ERROR");
+    //     }
```
Comment (Dev-AkshatGupta) [inline src/pages/authentication/Signup.jsx:44]: Should look for such commented  code and delete it

**Review by Dev-AkshatGupta — APPROVED** [review]
Few suggestions are given after that you are good to go and merge your code

File: src/pages/authentication/Signup.jsx (hunk 3 above)
Comment (nikhiljugale007) [reply src/pages/authentication/Signup.jsx:44]: I am going to use that code once i implement api calls. That why i kept it.

File: src/components/Footer/Footer.css (hunk 2 above)
Comment (nikhiljugale007) [reply src/components/Footer/Footer.css]: Removed

**Pull request merged**